hw/ip/i2s/data/i2s.hjson

```diff
@@ -109,7 +109,7 @@
       hwext:  "true"
       hwre:   "true"
       fields: [
-        { bits: "31:0", name: "RXDATA", desc: "latest rx data if DATA_READY flag is set", hwaccess: "hrw"}
```

Comment:
why?

Reply:
I forgot to reset I2S files a modified. I'll revert this.
I did this to understand how the window-workaround worked.
(For information, it may be useless to add this.)